test-assets/test-projects/AntlrGeneratedFiles/Test.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Antlr4.CodeGenerator" Version="4.6.4" />
```

Comment:
:bulb: Starting with 4.6.5-beta001 (already released), it is possible to eliminate the Java runtime requirement from the ANTLR build. If you don't currently require a Java installation for some other purpose, it might make sense to update the version and use the new code generator as described in the release notes:

https://github.com/tunnelvisionlabs/antlr4cs/releases/tag/v4.6.5-beta001

Reply:
FWIW, I haven't seen a Java installation requirement. I was able to get a design-time build of this without having Java installed.